src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/DataObjectHandler.php

```diff
@@ -135,11 +138,12 @@ public function getAllObjects()
      * Convert the parser output into a collection of EntityDataObjects
      *
      * @param string[] $parserOutput Primitive array output from the Magento parser.
+     * @param boolean  $validateName
      * @return EntityDataObject[]
      * @throws XmlException
-     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     * @SuppressWarnings(PHPMD)
```

Comment:
Can we suppress individual PHPMD warnings? It helps us know the potential problem and refactor in the future

Reply:
fixed